cmd/uplink/README.md

```diff
@@ -6,6 +6,8 @@ Usage:
 
 First make an identity:
 ```
+git clone https://github.com/storj/storj
+cd storj
```

Comment:
It could also be

`go get storj.io/storj/cmd/identity`

and then the go install will work as expected

However, who created this document assumed first to follow the instructions on the main README so that could be the reason why it's assumed that `storj.io/storj` is already in the GOPATH or the repo is already cloned in a local directory.